packages/DatePicker/src/components/Calendar/Calendar.styles.js

```diff
@@ -34,6 +37,9 @@ const iconButtonStyles = css`
 `;
 
 const calendarStyles = css`
+  height: ${calendarHeight}px;
```

Comment:
This seems to introduce a gap for most months:
![Screen Shot 2019-10-25 at 3 21 45 PM](https://user-images.githubusercontent.com/14944896/67607988-3f618e80-f73b-11e9-84c0-650b5e2363e9.png)

Reply:
Yes, as I have mentioned in the description. That is the negative of this solution